### ISSUE TITLE
iOS - `images` fields in Content.json generated in random order

### ISSUE THREAD

**Issue by bencergazda** [dwmkerr/app-icon#67]
This makes unwanted changes in the `AppIcon.appiconset/Content.json` file every time after running the script, even if we're running it against the same source icon.

run # 1:
```
{
  "images": [
    {
      "size": "40x40",
      "idiom": "iphone",
      "scale": "2x",
      "filename": "iphone-40x40-2x.png"
    },
    {
      "size": "29x29",
      "idiom": "iphone",
      "scale": "3x",
      "filename": "iphone-29x29-3x.png"
    },
    {
      "size": "29x29",
      "idiom": "iphone",
      "scale": "2x",
      "filename": "iphone-29x29-2x.png"
    },
    {
      "size": "29x29",
      "idiom": "iphone",
      "scale": "1x",
      "filename": "iphone-29x29-1x.png"
    },
    {
      "size": "20x20",
      "idiom": "ipad",
      "scale": "2x",
      "filename": "ipad-20x20-2x.png"
    },
    {
      "size": "20x20",
      "idiom": "iphone",
      "scale": "3x",
      "filename": "iphone-20x20-3x.png"
    },
    {
      "size": "20x20",
      "idiom": "iphone",
      "scale": "2x",
      "filename": "iphone-20x20-2x.png"
    },
    {
      "size": "20x20",
      "idiom": "ipad",
      "scale": "1x",
      "filename": "ipad-20x20-1x.png"
    },
    {
      "size": "40x40",
      "idiom": "iphone",
      "scale": "3x",
      "filename": "iphone-40x40-3x.png"
    },
    {
      "size": "57x57",
      "idiom": "iphone",
      "scale": "2x",
      "filename": "iphone-57x57-2x.png"
    },
    {
      "size": "60x60",
      "idiom": "iphone",
      "scale": "2x",
      "filename": "iphone-60x60-2x.png"
    },
    {
      "size": "57x57",
      "idiom": "iphone",
      "scale": "1x",
      "filename": "iphone-57x57-1x.png"
    },
    {
      "size": "60x60",
      "idiom": "iphone",
      "scale": "3x",
      "filename": "iphone-60x60-3x.png"
    },
    {
      "size": "72x72",
      "idiom": "ipad",
      "scale": "2x",
      "filename": "ipad-72x72-2x.png"
    },
    {
      "size": "83.5x83.5",
      "idiom": "ipad",
      "scale": "2x",
      "filename": "ipad-83.5x83.5-2x.png"
    },
    {
      "size": "29x29",
      "idiom": "ipad",
      "scale": "2x",
      "filename": "ipad-29x29-2x.png"
    },
    {
      "size": "50x50",
      "idiom": "ipad",
      "scale": "2x",
      "filename": "ipad-50x50-2x.png"
    },
    {
      "size": "76x76",
      "idiom": "ipad",
      "scale": "1x",
      "filename": "ipad-76x76-1x.png"
    },
    {
      "size": "29x29",
      "idiom": "ipad",
      "scale": "1x",
      "filename": "ipad-29x29-1x.png"
    },
    {
      "size": "72x72",
      "idiom": "ipad",
      "scale": "1x",
      "filename": "ipad-72x72-1x.png"
    },
    {
      "size": "40x40",
      "idiom": "ipad",
      "scale": "1x",
      "filename": "ipad-40x40-1x.png"
    },
    {
      "size": "76x76",
      "idiom": "ipad",
      "scale": "2x",
      "filename": "ipad-76x76-2x.png"
    },
    {
      "size": "50x50",
      "idiom": "ipad",
      "scale": "1x",
      "filename": "ipad-50x50-1x.png"
    },
    {
      "size": "40x40",
      "idiom": "ipad",
      "scale": "2x",
      "filename": "ipad-40x40-2x.png"
    },
    {
      "size": "1024x1024",
      "idiom": "ios-marketing",
      "scale": "1x",
      "filename": "ios-marketing-1024x1024-1x.png"
    }
  ],
  "info": {
    "version": 1,
    "author": "xcode"
  }
}
```

run # 2
```
{
  "images": [
    {
      "size": "40x40",
      "idiom": "ipad",
      "scale": "1x",
      "filename": "ipad-40x40-1x.png"
    },
    {
      "size": "29x29",
      "idiom": "ipad",
      "scale": "2x",
      "filename": "ipad-29x29-2x.png"
    },
    {
      "size": "29x29",
      "idiom": "ipad",
      "scale": "1x",
      "filename": "ipad-29x29-1x.png"
    },
    {
      "size": "60x60",
      "idiom": "iphone",
      "scale": "3x",
      "filename": "iphone-60x60-3x.png"
    },
    {
      "size": "60x60",
      "idiom": "iphone",
      "scale": "2x",
      "filename": "iphone-60x60-2x.png"
    },
    {
      "size": "57x57",
      "idiom": "iphone",
      "scale": "2x",
      "filename": "iphone-57x57-2x.png"
    },
    {
      "size": "57x57",
      "idiom": "iphone",
      "scale": "1x",
      "filename": "iphone-57x57-1x.png"
    },
    {
      "size": "40x40",
      "idiom": "iphone",
      "scale": "3x",
      "filename": "iphone-40x40-3x.png"
    },
    {
      "size": "40x40",
      "idiom": "iphone",
      "scale": "2x",
      "filename": "iphone-40x40-2x.png"
    },
    {
      "size": "29x29",
      "idiom": "iphone",
      "scale": "3x",
      "filename": "iphone-29x29-3x.png"
    },
    {
      "size": "29x29",
      "idiom": "iphone",
      "scale": "2x",
      "filename": "iphone-29x29-2x.png"
    },
    {
      "size": "29x29",
      "idiom": "iphone",
      "scale": "1x",
      "filename": "iphone-29x29-1x.png"
    },
    {
      "size": "20x20",
      "idiom": "ipad",
      "scale": "2x",
      "filename": "ipad-20x20-2x.png"
    },
    {
      "size": "20x20",
      "idiom": "iphone",
      "scale": "3x",
      "filename": "iphone-20x20-3x.png"
    },
    {
      "size": "20x20",
      "idiom": "iphone",
      "scale": "2x",
      "filename": "iphone-20x20-2x.png"
    },
    {
      "size": "20x20",
      "idiom": "ipad",
      "scale": "1x",
      "filename": "ipad-20x20-1x.png"
    },
    {
      "size": "40x40",
      "idiom": "ipad",
      "scale": "2x",
      "filename": "ipad-40x40-2x.png"
    },
    {
      "size": "76x76",
      "idiom": "ipad",
      "scale": "2x",
      "filename": "ipad-76x76-2x.png"
    },
    {
      "size": "50x50",
      "idiom": "ipad",
      "scale": "2x",
      "filename": "ipad-50x50-2x.png"
    },
    {
      "size": "72x72",
      "idiom": "ipad",
      "scale": "2x",
      "filename": "ipad-72x72-2x.png"
    },
    {
      "size": "50x50",
      "idiom": "ipad",
      "scale": "1x",
      "filename": "ipad-50x50-1x.png"
    },
    {
      "size": "83.5x83.5",
      "idiom": "ipad",
      "scale": "2x",
      "filename": "ipad-83.5x83.5-2x.png"
    },
    {
      "size": "76x76",
      "idiom": "ipad",
      "scale": "1x",
      "filename": "ipad-76x76-1x.png"
    },
    {
      "size": "72x72",
      "idiom": "ipad",
      "scale": "1x",
      "filename": "ipad-72x72-1x.png"
    },
    {
      "size": "1024x1024",
      "idiom": "ios-marketing",
      "scale": "1x",
      "filename": "ios-marketing-1024x1024-1x.png"
    }
  ],
  "info": {
    "version": 1,
    "author": "xcode"
  }
}
```

All the contents are the same, except of that in an other order.

**Comment by dwmkerr** [comment]
@bencergazda if you get a minute can you try using the branch `feat/order-icon-output`? I've tested and this seems to resolve the issue, but would be great if you can check too. If it works, I'll merge and cut a new release!

**Comment by bencergazda** [comment]
@dwmkerr Thanks for the feedback. This indeed fixes the issue!

**Comment by dwmkerr** [comment]
Done! Published as v0.6.4 :)
